source/index.md

```diff
@@ -1,18 +1,54 @@
-# Welcome to the Wootric Docs.
-
-Here is a list of the docs to integrate and use Wootric. It should not take more than 15 minutes. :-)
-<b>
-<br><br>
-* <a href="/javascript" style="text-decoration:none;color:#000">Install Wootric using Javascript</a>
-<br><br>
-* <a href="/segment" style="text-decoration:none;color:#000">Install Wootric using Segment.com</a>
-<br><br>
-* <a href="/ios" style="text-decoration:none;color:#000">Install Wootric using iOS SDK</a>
-<br><br>
-* <a href="/android" style="text-decoration:none;color:#000">Install Wootric using Android SDK</a>
-<br><br>
-* <a href="/api" style="text-decoration:none;color:#000">REST API</a>
-<br><br>
-* <a href="/webhooks" style="text-decoration:none;color:#000">Outgoing Webhooks</a>
-<br><br>
-</b>
+<div class="wootric-doc-landing">
+    <div class="wootric-doc-header">
+        <div class="wootric-doc-header__main">
+            Welcome to the Wootric documentation page!
+        </div>
+        <div class="wootric-doc-header__sub">
+            Select a documentation to get started
+        </div>
+    </div>
+    <div class="wootric-doc-landing-content">
+        <a class="wootric-doc-landing-content__type

```

Comment:
You are placing a block element inside an inline element. This is not ideal. Browsers won't tell you anything but keeping a valid markup gives you brownie points. Up to you if you want to change it or not (you can just set the `a` as `inline-block`)